noxfiles/dev_nox.py

```diff
@@ -29,10 +29,10 @@ def dev(session: nox.Session) -> None:
 
 @nox.session()
 def dev_with_worker(session: nox.Session) -> None:
-    """Spin up the entire application and open a development shell."""
+    """Spin up the entire application with the celery worker in a separate container."""
     build(session, "dev")
     session.notify("teardown")
-    session.run("docker-compose", "up", "worker", "--wait", external=True)
+    session.run("docker-compose", "up", "--detach", "worker", external=True)
```

Comment:
using the "--wait" flag properly: `session.run("docker-compose", "up", "--wait", "worker", external=True)` resulted in fidesops container never starting. This is because the `wait` flag will block until the current container exists. Celery container should not exit.